plugins/VersionUpgrade/VersionUpgrade460to462/VersionUpgrade460to462.py

```diff
@@ -80,8 +80,10 @@ def upgradeExtruderInstanceContainer(self, serialized: str, filename: str) -> Tu
             if parser["general"]["definition"] == "deltacomb_extruder_0":
                 parser["general"]["definition"] = "deltacomb_base_extruder_0"
             elif parser["general"]["definition"] == "deltacomb_extruder_1":  # Split up the second Deltacomb extruder into 3, creating an extra two extruders.
-                parser_e2 = copy.deepcopy(parser)
-                parser_e3 = copy.deepcopy(parser)
+                parser_e2 = configparser.ConfigParser(interpolation=None)
+                parser_e3 = configparser.ConfigParser(interpolation=None)
```

Comment:
Code style: spaces around binary operators. I'll change this myself to reduce ping-pong.